administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -158,7 +164,6 @@ COM_LOCALISE_PURGE="Purge Database"
 COM_LOCALISE_PURGE_FAILED="Failed to purge the localise table."
 COM_LOCALISE_PURGE_SUCCESS="The localise table has been emptied successfully."
 COM_LOCALISE_REFORMAT="Reformat j3 languages"
-COM_LOCALISE_REFORMAT_EXTENSION_SUCCESS="The \"%1$s\" ini files have been reformatted." ; The variable is the path to the extension
```

Comment:
Why delete this?

Reply:
Utch, by mistake.
Seems was using an not updated language file